.gitignore

```
@@ -1,4 +1,6 @@
 build
 .vs
 .idea
+Debug
```

Почему с большой буквы?
```suggestion
debug
```